exotic/exotic.py

```diff
@@ -1938,6 +1978,8 @@ def main():
 
                 # CALS
                 imageData = apply_cals(imageData, generalDark, generalBias, generalFlat, i)
+                # Demosaic, if needed
```

Comment:
How does it know "if needed"?

Reply:
@mikeprimm - are you missing an if statement here to trigger if the data needs to be demosaic-ed?

Reply:
no - like apply_cals just before it, the function implementation just responds to the fact that the input parameters are None when this particular type of calibration is being called for.